src/main/kotlin/net/atos/zac/smartdocuments/SmartDocumentsTemplatesService.kt

```diff
@@ -124,4 +125,94 @@ class SmartDocumentsTemplatesService @Inject constructor(
                 )
         ).resultList.toSet().toRestSmartDocumentsTemplateGroup()
     }
+
+    /**
+     * Get the information object type UUID for a pair of group-template in a zaakafhandelparameters
+     *
+     * @param zaakafhandelParametersUUID UUID of a zaakafhandelparameters
+     * @param templateGroupName name of a template group
+     * @param templateName name of a template under the group
+     * @return information object type UUID associated with this pair
+     */
+    fun getInformationObjectTypeUUID(
+        zaakafhandelParametersUUID: UUID,
+        templateGroupId: String,
+        templateId: String
+    ): UUID {
+        LOG.info {
```

Comment:
Pretty high logging level for this kind of normal functionality? Maybe make it into `fine`? Or do you want to see these log msgs on TEST for the time being and later on decrease the log level?